src/Components/CurrWeatherCard/CurrWeatherCard.jsx

```diff
@@ -37,8 +43,42 @@ export default function CurrWeatherCard({ locationData, currWeather }) {
         <img width="120" src={getWeatherSymbol(currWeather.current.symbol)} />
```

Comment:
i noticed you use everywhere `currWeather.current`, why not just pass `currWeather.current` as prop and make code a bit cleaner?